gothemis/session/session.go

```diff
@@ -213,12 +226,12 @@ func (ss *SecureSession) Unwrap(data []byte) ([]byte, bool, error) {
 	case (C.GOTHEMIS_SUCCESS == res) && (0 == outLen):
 		return nil, false, nil
 	case (C.GOTHEMIS_SSESSION_GET_PUB_FOR_ID_ERROR == res):
-		return nil, false, errors.NewCallbackError("Failed to get unwraped size (get_public_key_by_id callback error)")
+		return nil, false, ErrCallbackGetUnwrappedSize
```

Comment:
Well... this error isn't about size. THEMIS_SSESSION_GET_PUB_FOR_ID_CALLBACK_ERROR is returned when user-provided `GetPublicKeyForId` fails to retrieve a public key. I guess this should be `ErrNoPublicKey` or something.